README.rst

```diff
@@ -2,33 +2,33 @@
 Django Carrots tutorials
 ========================
 
-Tutorials walking new programmers through the process of building
-their first website in Python and Django.
+Le tutoriel Django Carrots présente pas-à-pas aux débutants la manière
+de créer son premier site internet avec Python et Django
 
-Loose outline
+En quelques mots:
 
-#. Installing Python and setting up development environment
-#. Python basics
-#. "Snakes!" online competition
-#. Creating a polling application in Django
-#. Deploying the application
+#. Installer Python et mettre en place son environnement de développement
+#. Les bases de Python
+#. "Snakes!" competition en ligne
+#. Mettre en place une appli de sondage en Django
```

Comment:
"Mettre en place" is more "Install". I would suggest a verb like "Créer" or "Développer".